osquery/tables/system/python_packages.cpp

```diff
@@ -84,12 +84,43 @@ void genPackage(const std::string& path, Row& r, Logger& logger) {
   }
 }
 
+Status stdListDirectoriesInDirectory(const fs::path& path,
```

Comment:
Is this useful enough we should put in filesystem.cpp?

Reply:
My idea was to release this new logic only and then wait a couple of releases to see how it behaves. I can benchmark it on different platforms before deciding on using it on filesystems.cpp